dist/commands/head.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var interfacer = require('./../util/interfacer');
+var fs = require('fs');
+var fsAutocomplete = require('vorpal-autocomplete-fs');
+
+var head = {
+  exec: function exec(args, options) {
+    var self = this;
+    options = options || {};
+
+    options.argsType = args.stdin === undefined ? 'files' : 'stdin';
+    options.n = options.n === undefined ? 10 : options.n;
+
+    if (options.n < 1) {
+      self.log('Option n must be a positive integer.');
```

Comment:
I see your point. I think supporting negative options wouldn't necessarily be a bad move, but it seems like it's not necessary according to this standard.

I believe 0 is not a positive integer. So it's up to you if you want to support it or not, just like with negative values :+1: